docs/howtos/read_nonlinear_data.rst

```diff
@@ -0,0 +1,46 @@
+====================================================
+Reading and plotting nonlinear outputs
+====================================================
+
+This is a step-by-step guide on how to use ``pyrokinetics`` to read the ouput of a nonlinear gyrokinetic simulation and create some simple plots. In this guide, we will use the example nonlinear output from CGYRO. The same ideas apply for  any gyrokinetic codes supported in ``pyrokinetics``,
+
+Let's first import ``pyrokinetics`` and define our nonlinear input file.  
+
+.. code-block:: python
+
+   >>> from pyrokinetics import Pyro, template_dir
+   >>> gk_file = template_dir / "outputs/CGYRO_nonlinear/input.cgyro"
+
+The gyrokinetic file ``input.cgyro`` is our input file template where we set all the extra flags that have been used in the simulation.  
+
+We can then read the nonlinear simulation into a dataset using ``pyrokinetics``.
+
+.. code-block:: python 
+
+   >>> pyro.load_gk_output(load_moments=True, load_fluxes=True, load_fields=True)
```

Comment:
This is probably obvious but should we state what these kwargs do? Also which ones are on by default